react/react_intro/README.md

```diff
@@ -144,7 +144,7 @@ From the JSX in the return statement, we see another interesting bit of syntax:
 
 Much like how string interpolation is used to embed variables in strings, JSX uses `{}` to embed expressions.  Here, `logo` is the string `./logo.svg`
 
-[React](https://reactjs.org/docs/introducing-jsx.html) gives the following example of how JSX can be used:
+[React](https://reactjs.org/docs/introducing-jsx.html) gives the following example of how JSX can be used. In this examlpe below we see `const element` being assigned a JSX value: 
```

Comment:
Is there a better example we can use to illustrate this point? I'm concerned that this example is showing `ReactDOM.render()` which is really something we don't need to get into right now. This seems like a rabbit hole that we can avoid. If we're using create react app, they really don't need to know about `ReactDOM.render()` right now in order to get their apps running.